### DISCUSSION TITLE
How to create a branch for contributing to PYG

### DISCUSSION THREAD

**Discussion by AlmogDavid** [pyg-team/pytorch_geometric#9729] [Q&A] [answered]
Hi, I'm trying to contribute a module to PYG and I followed the contribution guide. It seems like I don't have permisssion to open a branch. How can I open a branch so I will be able to do a merge request?

Thanks.

**Comment by akihironitta** [comment, marked as answer]
I'd definitely recommend taking a look at https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a. Looking forward to your contributions!
